webapplication/notificator/templates/email_notification.html

```diff
@@ -0,0 +1,2 @@
+Hello {{ username }}, your request of component {{ component }} have been processed.
+Result for timeperiod from {{ startdate }} to {{ enddate }} is published.
```

Comment:
This should include links to all results associated with request. As well request status (failed, completed, etc.)